skt/executable.py

```diff
@@ -190,7 +190,7 @@ def cmd_merge(cfg):
     """
     global retcode
     # Counter merge patch following:
-    # id[0]:  counter of merge-ref option.
+    # idx[0]: counter of merge-ref option.
```

Comment:
"Fix typo in the comment" would be better to use in this commit message, what do you say?